controllers/toplevel/search.js

```diff
@@ -0,0 +1,19 @@
+const querystring = require('querystring');
+const { noCache } = require('../../middleware/cached');
+const { trackPageview } = require('../../modules/analytics');
+
+function init({ router, routeConfig }) {
+    const queryBase = 'https://www.google.co.uk/search?q=site%3Abiglotteryfund.org.uk';
```

Comment:
Do you think we should use Welsh-language Google for the CY locale?

Reply:
Good shout